pkg/ccr/base/spec.go

```diff
@@ -1238,6 +1238,8 @@ func (s *Spec) LightningSchemaChange(srcDatabase, tableAlias string, lightningSc
 	sql = strings.Replace(sql, "REPLACE_IF_NOT_NULL NULL DEFAULT \"null\"",
 		"REPLACE_IF_NOT_NULL NULL DEFAULT NULL", 1)
 
+	sql = strings.ReplaceAll(sql, "\"CURRENT_TIMESTAMP\"", "CURRENT_TIMESTAMP")
```

Comment:
1. The CREATE_TABLE binlog should fix this issue too.
2. consider replacing with `DEFAULT "CURRENT_TIMESTAMP"` to `DEFAULT CURRENT_TIMESTAMP`
3. please add some test cases.